code-studio/src/js/initApp/project.js

```diff
@@ -129,6 +129,13 @@ var projects = module.exports = {
   },
 
   /**
+   * Whether this project uses Firebase for data storage.
+   */
+  getUseFirebase: function () {
+    return current.useFirebase;
+  },
```

Comment:
Though I don't feel strongly about it, this name sounds a little awkward to me. What about `usesFirebase()`?